src/date-util/date-util.type.ts

```diff
@@ -4,3 +4,4 @@ import { Iso8601FormatType } from './date-util-base.type';
 export type CalcDatetimeOpts = Partial<DatetimeProperties>;
 export type DatetimeFormatOpts = Partial<FormatOpts<string>>;
 export type IsoDatetimeFormatOpts = Partial<FormatOpts<Iso8601FormatType>>;
+export type DateUnitType = 'seconds' | 'minutes' | 'hours';
```

Comment:
DatePropertyType을 그냥 써도 되지 않나요?
제가 DateUnitType을 얘기한 것을 반영하고자 하는 거라면
아래처럼 하고 DatePropertyType을 deprecated 처리하는 것이 어떨지?
```suggestion
export type DateUnitType = DatePropertyType;
```

Reply:
아 있는걸 몰랐네요 ;; 확인 감사합니다

Reply:
DatePropertyType을 사용하는걸로 할게요